[2024-12-04 05:50:11,845] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: KPIs_for_5_Minutes.fetch_master_Data 2024-12-04T05:45:00+00:00 [queued]>
[2024-12-04 05:50:11,904] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: KPIs_for_5_Minutes.fetch_master_Data 2024-12-04T05:45:00+00:00 [queued]>
[2024-12-04 05:50:11,905] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-12-04 05:50:11,906] {taskinstance.py:1088} INFO - Starting attempt 1 of 3
[2024-12-04 05:50:11,907] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-12-04 05:50:11,995] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): fetch_master_Data> on 2024-12-04T05:45:00+00:00
[2024-12-04 05:50:12,001] {standard_task_runner.py:52} INFO - Started process 1048 to run task
[2024-12-04 05:50:12,007] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'KPIs_for_5_Minutes', 'fetch_master_Data', '2024-12-04T05:45:00+00:00', '--job-id', '469', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/Kpi_dags/main.py', '--cfg-path', '/tmp/tmp8tqd0pmv', '--error-file', '/tmp/tmpzdvi1_g_']
[2024-12-04 05:50:12,009] {standard_task_runner.py:77} INFO - Job 469: Subtask fetch_master_Data
[2024-12-04 05:50:12,195] {logging_mixin.py:104} INFO - Running <TaskInstance: KPIs_for_5_Minutes.fetch_master_Data 2024-12-04T05:45:00+00:00 [running]> on host ca28183b9a80
[2024-12-04 05:50:12,542] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Rule Manager
AIRFLOW_CTX_DAG_ID=KPIs_for_5_Minutes
AIRFLOW_CTX_TASK_ID=fetch_master_Data
AIRFLOW_CTX_EXECUTION_DATE=2024-12-04T05:45:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-12-04T05:45:00+00:00
[2024-12-04 05:50:12,544] {main.py:98} INFO - Fetching data from master db and storing excel between 2024-12-04 05:45:00 and 2024-12-04 05:50:00
[2024-12-04 05:50:12,545] {main.py:23} INFO - Connecting to database...
[2024-12-04 05:50:12,581] {base.py:69} INFO - Using connection to: id: master_table_conn. Host: 10.10.18.92, Port: 5432, Schema: landing_zone, Login: Chakradhar, Password: ***, extra: {}
[2024-12-04 05:50:12,854] {main.py:26} INFO - Connected successfully...
[2024-12-04 05:50:12,959] {main.py:106} INFO - data fetched successfully from master db and size is (0, 8)
[2024-12-04 05:50:12,960] {main.py:107} INFO - Empty DataFrame
Columns: [ts, tag_obj_id, asset_id, value, created_by, updated_by, create_time, updated_time]
Index: []
[2024-12-04 05:50:12,975] {main.py:109} INFO - data stored successfully
[2024-12-04 05:50:12,977] {python.py:151} INFO - Done. Returned value was: None
[2024-12-04 05:50:13,041] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=KPIs_for_5_Minutes, task_id=fetch_master_Data, execution_date=20241204T054500, start_date=20241204T055011, end_date=20241204T055013
[2024-12-04 05:50:13,369] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-04 05:50:13,416] {local_task_job.py:151} INFO - Task exited with return code 0
